src/components/Player.js

```diff
@@ -19,28 +26,67 @@ const Player = ({currentSong, isPlaying, setIsPlaying}) => {
       setIsPlaying(!isPlaying);
     }
   };
+
+  //extract currentTime of song and duration
+  const timeUpdateHandler = (e) => {
+    const current = e.target.currentTime;
+    const duration = e.target.duration;
+    //object used to update state when song is playing
+    setSongInfo({...songInfo, currentTime: current, duration});
+  };
+
+  //format song time in minutes and seconds
+  const getTime = (time) => {
+    return (
+      Math.floor(time / 60) + ':' + ('0' + Math.floor(time % 60)).slice(-2)
+    );
+  };
+
+  //update state when the onChange event is ran
+  const dragHandler = (e) => {
+    audioRef.current.currentTime = e.target.value;
+    setSongInfo({...songInfo, currentTime: e.target.value});
+  };
+
   return (
     <div className="player">
       <div className="time-control">
-        <p>Start Time</p>
-        <input type="range" />
-        <p>End Time</p>
+        <p>{getTime(songInfo.currentTime)}</p>
```

Comment:
No need for jsx to be spacing/line breaks as its still considered 'html', thus html styling applies. Only JS logic above the return statement on line 51 needs js formatting conventions applied